digitalpublicgoods/ottaa-project.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "OTTAA Project",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ottaa-project.github.io/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "No",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://ottaa-project.github.io/docs/welcome",
+      "https://ottaa-project.github.io/PictogramsPredictionsLibrary/docs/readme/",
```

Comment:
This link is not working for me, I think they might refer to this one -> https://ottaa-project.github.io/docs/Readme